packages/front-end/components/EventWebHooks/EventWebHookDetail/EventWebHookDetail.tsx

```diff
@@ -101,144 +146,168 @@ export const EventWebHookDetail: FC<EventWebHookDetailProps> = ({
-            Edit
-          </button>
-
-          <button
-            onClick={onTestWebhook}
-            className="btn btn-sm btn-outline-secondary mr-1"
-            disabled={state && state.type === "loading"}
-          >
-            <FaPaperPlane className="mr-1" />
-            Test
-          </button>
-
-          <DeleteButton
-            displayName={name}
-            onClick={onDelete}
-            outline={true}
-            className="btn-sm"
-            text="Delete"
-          />
+        <div className="ml-2">
+          {!lastRunAt ? (
+            <div className="text-muted">No runs</div>
+          ) : (
+            <div className="text-main d-flex align-items-center">
+              <b className="mr-1">Last run:</b> {datetime(lastRunAt)}
+              <span className="ml-2" style={{ fontSize: "1.5rem" }}>
+                {iconForState}
+              </span>
+            </div>
+          )}
         </div>
-      </div>
 
-      <h3 className="text-muted text-break font-weight-bold">{url}</h3>
+        <div className="ml-2 d-flex align-items-center">
+          <div className="text-main">
+            <b>Secret:</b>
+          </div>
+          <span className="ml-1">
+            <code className="text-main text-break">{signingKey}</code>
+          </span>
+
+          <span className="ml-2">
+            {copySupported ? (
+              <button
+                className="btn p-0 pb-1"
+                onClick={() => performCopy(signingKey)}
+              >
+                <span className="text-main" style={{ fontSize: "1.1rem" }}>
+                  {copySuccess ? (
+                    <HiOutlineClipboardCheck />
+                  ) : (
+                    <HiOutlineClipboard />
+                  )}
+                </span>
+              </button>
+            ) : null}
+          </span>
+        </div>
+      </div>
 
-      <div className="card mt-3 p-3">
+      <div className="card mt-3 p-3 p-4">
         <div className="row">
           <div className="col-xs-12 col-md-6">
-            <div className="d-flex font-weight-bold align-items-center">
-              {/* Last run state & date */}
-              <span className="mr-2" style={{ fontSize: "1.5rem" }}>
-                {iconForState}
-              </span>
-              {lastRunAt ? (
-                <span
-                  className={classNames("", {
-                    "text-success": lastState === "success",
-                    "text-danger": lastState === "error",
-                  })}
-                >
-                  Last run on {datetime(lastRunAt)}
-                </span>
-              ) : (
-                <span className="text-muted">
-                  This webhook has not yet run.
-                </span>
-              )}
+            <div className="align-items-center mt-2">
+              <span className="font-weight-bold">Events enabled</span>
+              <div className="mt-1">{displayedEvents(events)}</div>
```

Comment:
@romain-growthbook I definitely see the value in only showing up to 5 events on the `EventWebhookListItem` component, but on the detail page, it feels like we should display all of the events, not just up to 5. Thoughts?

Reply:
Cool. I remove the upper limit on this list. Thanks for the review!